frontend/actions/user.ts

```diff
@@ -49,9 +49,9 @@ interface UserInformationResponse {
   entity: UserInformation
 }
 
-export function useGetUserInformation(dn: string) {
+export function useGetUserInformation(dn: string, kind?: string) {
```

Comment:
Can we give `kind` a more specific type? I'm assuming this can be `'user'` or `'group'` but I'm not sure just by looking at it.

Reply:
This is no longer needed